recipes/xilinx-runtime/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/env/bash
+set -e
+
+mkdir -p $SRC_DIR/build-release
+
+cmake \
+  -S $SRC_DIR/src  \
```

Comment:
Add `$CMAKE_ARGS` to pick up our default arguments.
```suggestion
cmake \
  $CMAKE_ARGS \
  -S $SRC_DIR/src  \
```